packages/pds/tests/feed-generation.test.ts

```diff
@@ -347,6 +347,18 @@ describe('feed generation', () => {
     })
   })
 
+  describe('putNotifications', () => {
+    it('updates notifications', async () => {
```

Comment:
guessing this was just for internal testing but will wanna clean up before merge

Reply:
yes. I am still moving my tests over to a new file!